### DISCUSSION TITLE
Stolen Content

### DISCUSSION THREAD

**Discussion by ReallyBadDeveloper** [inwinter04/win11React_CN#25] [General]
This was obviously stolen from win11.blueedge.me. It wasn't even edited. Seriously?

**Comment by inwinter04** [comment]
> This was obviously stolen from win11.blueedge.me. It wasn't even edited. Seriously?

localization language of win11.blueedge.me. In fact, BlueEdge is fully aware of the existence of the project and the project has also marked the link of the original project. Similarly, the original project also has the link of the project.🧐
